tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1893,6 +1893,30 @@ def func(self, xp, a):
         return xp.arctan(a)
 
 
+@chainerx.testing.numpy_chainerx_array_equal(strides_check=False)
```

Comment:
A minor but it would be good if you could leave a comment in the code why the strides check must be skipped.

Reply:
@hvy Thanks for pointing out. On further inspection it is not required as we manually cast `numpy` output to expected `chainerx` outputs. This is done as `numpy` defaults to `float64` while `chainerx` to `float32`.

Have removed the `strides_check=False`.